parsers/nmparser/parse_run_details.go

```diff
@@ -31,27 +49,62 @@ func ParseRunDetails(lines []string) RunDetails {
 	covarianceTime := 0.0
 	functionEvaluations := int64(0)
 	significantDigits := 0.0
-	for _, line := range lines {
+	problemText := ""
+	modFile := "" // TODO, pass in from caller
+	estimationMethod := []string{}
+	dataSet := ""
+	numberOfPatients := int64(0)
+	numberOfObs := int64(0)
+	numberOfDataRecords := int64(0)
+	outputTable := ""
+
+	for i, line := range lines {
 		switch {
 		case strings.Contains(line, "1NONLINEAR MIXED EFFECTS MODEL PROGRAM (NONMEM) VERSION"):
 			nmversion = parseNMVersion(line)
 		case strings.Contains(line, "NO. OF FUNCTION EVALUATIONS USED"):
 			functionEvaluations, _ = strconv.ParseInt(replaceTrim(line, "NO. OF FUNCTION EVALUATIONS USED:"), 10, 64)
 		case strings.Contains(line, "NO. OF SIG. DIGITS IN FINAL EST.:"):
 			significantDigits, _ = strconv.ParseFloat(replaceTrim(line, "NO. OF SIG. DIGITS IN FINAL EST.:"), 64)
-		case strings.Contains(line, "Elapsed estimation time in seconds:"):
+		case strings.Contains(line, "Elapsed estimation"):
 			estimationTime = parseFinalTime(line)
 		case strings.Contains(line, "Elapsed covariance time in seconds:"):
 			covarianceTime = parseFinalTime(line)
+		case strings.Contains(line, "Elapsed postprocess time in seconds:"):
+			covarianceTime = parseFinalTime(line)
 		case strings.Contains(line, "Started"):
 			runStart = replaceTrim(line, "Started")
 		case strings.Contains(line, "Finished"):
 			runEnd = replaceTrim(line, "Finished")
+		case strings.Contains(line, "Stop Time:"):
+			if i+1 < len(lines) {
+				runEnd = lines[i+1]
+			}
+		case strings.Contains(line, "$PROB"):
+			problemText = replaceTrim(line, "$PROB")
+		case strings.Contains(line, "$ESTIMATION MAXEVAL"):
```

Comment:
we should parse this out elsewhere. There are a number of ways this will fail.

user can define shorthand `$EST`, you do not need to do MAXEVAL directly after `$ESTIMATION METHOD=1 MAXEVAL=9999`, etc. To abstract this, we should test for `$EST` then feed that entire line into a parseEstimationDetails type function so we can change that implementation as we go a little more easily

Reply:
OK.